Header header
float64 north
float64 east

float32 vel

uint32 action
